docassemble/AssemblyLine/al_document.py

```diff
@@ -43,25 +43,34 @@ def html_safe_str(the_string: str) -> str:
   """
   return re.sub( r'[^A-Za-z0-9]+', '_', the_string )
 
-def table_row( title, view_file:DAFile, download_file:DAFile=None, view_icon:str="eye", download_icon:str="download") -> str:
+#def table_row( title, view_file:DAFile, download_file:DAFile=None, view_icon:str="eye", download_icon:str="download") -> str:
+def table_row( title:str, buttons:list = []) -> str:
```

Comment:
```suggestion
def table_row( title:str, button_htmls:List[str] = []) -> str:
```
Just would make it a bit easier to understand at first glance.